site/content/en/docs/formats/synthia.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'SYNTHIA'
+linkTitle: 'SYNTHIA'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original SYNTHIA dataset is available
+[here](https://synthia-dataset.net).
+
+Supported annotation types:
+- `Masks`
+
+## Import SYNTHIA dataset
+
+A Datumaro project with a SYNTHIA source can be created in the following way:
+
+```bash
+datum create
+datum import --format synthia <path/to/dataset>
+```
+
+It is also possible to import the dataset using Python API:
+
+```python
+from datumaro.components.dataset import Dataset
+
+synthia_dataset = Dataset.import_from('<path/to/dataset>', 'synthia')
+```
+
+SYNTHIA dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+dataset/
+├── GT/
+│   └── LABELS/
+│       ├── Stereo_Left/
+│       │   ├── Omni_B/
+│       │   │   ├── 000000.png
+│       │   │   ├── 000001.png
+│       |   |   └── ...
+│       │   └── ...
+│       └── Stereo_Right/
+│           ├── Omni_B/
+│           │   ├── 000000.png
+│           │   └── 000001.png
+│           |   └── ...
+│           └── ...
+└── RGB/
+    ├── Stereo_Left/
+    │   ├── Omni_B/
+    │   │   ├── 000000.png
+    │   │   ├── 000001.png
+    |   |   └── ...
+    │   └── ...
+    └── Stereo_Right/
+        ├── Omni_B/
+        │   ├── 000000.png
+        │   └── 000001.png
+        |   └── ...
+        └── ...
+```
+
+- RGB folder containing standard RGB images used for training.
+- GT/LABELS folder containing containing png files (one per image).
+Annotations are given in two channels. The first channel contains
+the class of that pixel (see the table below). The second channel
+contains the unique ID of the instance for those objects
+that are dynamic (cars, pedestrians, etc.).
+
+Also present in the original dataset:
+- GT/COLOR folder containing png files (one per image).
+Annotations are given using a color representation.
+- Depth folder containing unsigned short images. Depth is encoded
+in any of the 3 channels in centimetres as an ushort.
+But this information can be obtained from the instance segmentation.
```

Comment:
I don't really understand this.

* How could depth be encoded in _any_ of the 3 channels? Are all 3 channels the same? Why are there 3 channels to begin with?
* How can depth information be obtained from instance segmentation?

Reply:
Three channels have the same value. I also don't understand why to store a three-channel image.

We can find out the color by the class id. There is a one-to-one compliance between them.

Reply:
Three channels have the same value. I also don't understand why to store a three-channel image.

We can find out the color by the class id. There is a one-to-one compliance between them.

Reply:
Is it documented anywhere officially? Otherwise, it can be related to PIL or OpenCV interpretation.

Reply:
> Three channels have the same value. I also don't understand why to store a three-channel image.

Weird, but OK.

_That said_, do we even need to document this at all if the extractor ignores this data anyway? IMHO, it would be enough to say something like "the original dataset also contains depth information, but Datumaro does not currently support it".

> We can find out the color by the class id. There is a one-to-one compliance between them.

I don't understand what you're talking about. My question was about obtaining _depth_, not color.

> Is it documented anywhere officially? Otherwise, it can be related to PIL or OpenCV interpretation, we've already faced it in some cases

No, I actually went and checked a random depth image myself, and it was indeed RGB with 16-bit components (I even looked at the hex dump to be sure).